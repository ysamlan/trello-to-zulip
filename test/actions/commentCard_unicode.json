{
  "data": {
    "board": {
      "id": "b1idb1idb1idb1idb1idb1id",
      "name": "Board One",
      "shortLink": "b1slb1sl"
    },
    "card": {
      "id": "cidcidcidcidcidcidcidcid",
      "idShort": 1,
      "name": "Card Name",
      "shortLink": "cslcslss"
    },
    "text": "Frosty was a \u2603 and a very happy one he was."
  },
  "date": "2013-11-03T20:52:58.549Z",
  "id": "actidactidactidactidacti",
  "idMemberCreator": "mcidmcidmcidmcidmcidmcid",
  "memberCreator": {
    "avatarHash": "mcahmcahmcahmcahmcahmcahmcahmcah",
    "fullName": "Member Creator Full Name",
    "id": "mcidmcidmcidmcidmcidmcid",
    "initials": "MCFN",
    "username": "mcfname"
  },
  "type": "commentCard"
}
