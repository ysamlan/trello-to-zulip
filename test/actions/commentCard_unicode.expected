Member Creator Full Name commented on card [Card Name](https://trello.com/c/cidcidcidcidcidcidcidcid) 
>Frosty was a â˜ƒ and a very happy one he was.